src/gui/screen_printing.cpp

```diff
@@ -425,7 +425,7 @@ static void update_remaining_time(screen_t *screen, time_t rawtime) {
             snprintf(array.data(), array.size(), "%im", timeinfo->tm_min);
         }
     } else
-        strcpy_P(array.data(), PSTR("N/A"));
+        strlcpy_P(array.data(), PSTR("N/A"), array.size());
```

Comment:
WTF? PSTR and strlcpy_P are 8bit AVR relicts because of different instructions reading from RAM and program memory